memcached.c

```diff
@@ -4926,6 +5012,13 @@ static void process_command(conn *c, char *command) {
 #ifdef EXTSTORE
     } else if (ntokens >= 3 && strcmp(tokens[COMMAND_TOKEN].value, "extstore") == 0) {
         process_extstore_command(c, tokens, ntokens);
+#endif
+#ifdef TLS
+    } else if (ntokens >= 1 && ntokens <= 3 && (strcmp(tokens[COMMAND_TOKEN].value, "refresh_certs") == 0)) {
+        set_noreply_maybe(c, tokens, ntokens);
+        refresh_certificates();
```

Comment:
refresh_certificates() prints to stderr. Should create a string and return to the caller, which is then copied or freed.